bazel/external/llvm.patch

```diff
@@ -0,0 +1,25 @@
+# Workaround for Envoy's CMAKE_BUILD_TYPE=Bazel.
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -247,7 +247,7 @@
+ string(TOUPPER "${CMAKE_BUILD_TYPE}" uppercase_CMAKE_BUILD_TYPE)
+ 
+ if (CMAKE_BUILD_TYPE AND
+-    NOT uppercase_CMAKE_BUILD_TYPE MATCHES "^(DEBUG|RELEASE|RELWITHDEBINFO|MINSIZEREL)$")
++    NOT uppercase_CMAKE_BUILD_TYPE MATCHES "^(DEBUG|RELEASE|RELWITHDEBINFO|MINSIZEREL|BAZEL)$")
```

Comment:
This is patched, but never used.

I think that this isn't needed anymore (it was originally added to switch between `DEBUG` and `RELEASE` build types based on `-c dbg` and `-c opt`, but I think `rules_foreign_cc` support that nowadays, so this isn't needed anymore).

Reply:
Thanks, it works.